packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -116,7 +116,7 @@
     "grammars": [
       {
         "language": "html",
-        "scopeName": "text.html.basic",
+        "scopeName": "aura.html",
```

Comment:
By making this name have 2 parts instead of 3 it keeps it from overriding the styles incorrectly for php html.